doc/source/CustomizingTheWorkflow/ConfigWorkflow.rst

```diff
@@ -141,9 +152,36 @@ Entities are constants that can be referred to throughout the workflow using the
 ``NPROCS_ANALYSIS:`` (Default: "6")
    Number of processors for the analysis task. 
 
-``NPROCS_FORECAST:`` (Default: "7")
-   Number of processors for the forecast task. 
+``DT_ATMOS:`` (Default: "900")
+   Time integration step of the atmospheric component of the UFS weather model (in seconds).
+   
+``DT_RUNSEQ:`` (Default: "6")
+   Time interval of run sequence (coupling interval) between the model components of the UFS weather model (in seconds).
+
+``NPROCS_FORECAST:`` (Default: "26")
+   Total number of processes for the FORECAST task.
+
+``NPROCS_FORECAST_ATM:`` (Default: "12")
+   Number of processes for the atmospheric model component (DATM) in the FORECAST task.
+
+``NPROCS_FORECAST_LND:`` (Default: "12")
+   Number of processes for the land model component (Noah-MP) in the FORECAST task.
```

Comment:
Any idea why these only add up to 24 instead of the 26 listed in `NPROCS_FORECAST`? I'm curious how they're calculated...